### PR TITLE
Updated openssl to 1.1.1i

### PR DESCRIPTION
**Pull request by randomshinichi**
How can I make Homebrew use its own Perl 5.22? And what should one do about those sha256 sums for bottles that don't run on PPC?

### CODE REVIEW

File: Library/Formula/openssl.rb [hunk 1]
```diff
@@ -72,8 +66,8 @@ def install
         system "make", "clean"
       end
 
-      ENV.deparallelize
-      system "perl", "./Configure", *(configure_args + arch_args[arch])
+      # ENV.deparallelize
+      system "/usr/local/Cellar/perl/5.22.0/bin/perl", "./Configure", *(configure_args + arch_args[arch])
```
Comment (carlocab) [inline Library/Formula/openssl.rb:70]:
```suggestion
      system "#{Formula["perl"].opt_bin}/perl", "./Configure", *(configure_args + arch_args[arch])
```
That said, I don't think this should be needed, as `brew` should already be putting Homebrew `perl` ahead of system `perl` in your `PATH` during the build.

File: Library/Formula/openssl.rb [hunk 2]
```diff
@@ -72,7 +66,13 @@ def install
         system "make", "clean"
       end
 
-      ENV.deparallelize
+
+      # This ensures where Homebrew's Perl is needed the Cellar path isn't
+      # hardcoded into OpenSSL's scripts, causing them to break every Perl update.
+      # Whilst our env points to opt_bin, by default OpenSSL resolves the symlink.
+      ENV["PERL"] = Formula["perl"].opt_bin/"perl" if which("perl") == Formula["perl"].opt_bin/"perl"
```
Comment (carlocab) [inline Library/Formula/openssl.rb:73]: Isn't your `if` statement always true here given that `perl` is a build dependency?

File: Library/Formula/openssl.rb (hunk 1 above)
Comment (randomshinichi) [reply Library/Formula/openssl.rb]: No, that didn't happen. It tried to use Perl 5.8. Remember this is Tigerbrew, not Homebrew

Comment (carlocab) [reply Library/Formula/openssl.rb]: I'm aware. I thought Tigerbrew also properly managed `PATH` during buildtime.

**Comment by sevan** [comment]
1.1.1v is now available via openssl formula.

**Pull request closed without merging**